src/boot/init.ts

```diff
@@ -15,10 +16,10 @@ declare module "@vue/runtime-core" {
 }
 
 export default boot(async ({ app }) => {
-  const initActions = [
-    globalState.init()
-  ]
-  await Promise.all(initActions)
+  await appConfig.init()
+  const { nodeName } = (await appConfig.read());
```

Comment:
what if nodeName was not set in the config (first time start of the application)?

Reply:
it will use initial value - empty string